### PR TITLE
feat(plugin-chart-echarts): add orderby on Radar chart

### PR DESCRIPTION
**Pull request by xiezhongfu**
add SORT BY metrics to Radar chart to avoid random values.

### before
radar sort by metrics before

### after
radar sort by metrics after 2

### CODE REVIEW

**Comment by zhaoyongjie** [comment]
Thanks for the fix!

How do we face sort by specific metric? for instance, order by max(value).

image

**Comment by xiezhongfu** [comment]
> Thanks for the fix!
> 
> How do we face sort by specific metric? for instance, order by max(value).
> 
> image

sort by specific metric.
refer:  https://github.com/apache-superset/superset-ui/blob/master/plugins/plugin-chart-table/src/buildQuery.ts#L68
> * default to ordering by first metric in descending order when no "sort by" metric is set
> * orverride orderby with timeseries metric
em. I do not know why `timeseries metric` is time-series.

**Comment by wjl520** [comment]
how to add radar chart on the dashboard, namely i see nothing when i choose a visualization type for radar chart. Looking forward to your reply. thx.

**Comment by xiezhongfu** [comment]
> how to add radar chart on the dashboard, namely i see nothing when i choose a visualization type for radar chart. Looking forward to your reply. thx.

Please let me know more about your issue. You can show something more about error.

**Comment by zhaoyongjie** [comment]
> how to add radar chart on the dashboard, namely i see nothing when i choose a visualization type for radar chart. Looking forward to your reply. thx.

Hi, the Radar chart is already available on the master branch.

File: plugins/plugin-chart-echarts/src/Radar/controlPanel.tsx [hunk 1]
```diff
@@ -70,6 +70,7 @@ const config: ControlPanelConfig = {
       controlSetRows: [
         ['groupby'],
         ['metrics'],
+        ['timeseries_limit_metric'],
```
Comment (zhaoyongjie) [inline plugins/plugin-chart-echarts/src/Radar/controlPanel.tsx:73]: I will file a new PR to add orderby shared-control. we don't need this control in the non-timeseries chart.

**Comment by junlincc** [comment]
> how to add radar chart on the dashboard, namely i see nothing when i choose a visualization type for radar chart. Looking forward to your reply. thx.

please file improvement or bug requests once you get to it, we are looking for ways to improve the chart. thank you :)

File: plugins/plugin-chart-echarts/src/Radar/buildQuery.ts [hunk 2]
```diff
@@ -16,14 +16,26 @@
...
 
 export default function buildQuery(formData: QueryFormData) {
-  const { metric, sort_by_metric } = formData;
-  return buildQueryContext(formData, baseQueryObject => [
-    {
-      ...baseQueryObject,
-      ...(sort_by_metric && { orderby: [[metric, false]] }),
-    },
-  ]);
+  const { orderby } = formData;
```
[the remark this answers is not in the document]
Comment (zhaoyongjie) [reply plugins/plugin-chart-echarts/src/Radar/buildQuery.ts:22]: You are right. revert change, use `timeseries_limit_metric` back. the `orderby` leave it to future new sortby control.

File: plugins/plugin-chart-echarts/src/Radar/controlPanel.tsx (hunk 1 above)
Comment (xiezhongfu) [reply plugins/plugin-chart-echarts/src/Radar/controlPanel.tsx:73]: refer:https://github.com/apache-superset/superset-ui/pull/1112#discussion_r636270729

**Review by zhaoyongjie — APPROVED** [review]
LGTM, Thanks for this.

**Pull request merged**